### ISSUE TITLE
CyKITv2 /Unity

### ISSUE THREAD

**Issue by XORGIN** [CymatiCorp/CyKITv2#13]
Hi
Do you think you will creat a Unity plugin? 
I question myself about the link with the two.  Epoc works only with 32bit? and Unity 64 bit?

thanks

**Comment by warrenarea** [comment]
yes, i have created a Unity3D plugin to connect as a socket server to the CyKIT generic TCP streaming... but i just haven't uploaded it anywhere yet.

the 32-bit or 64-bit shouldn't make much of a difference.

CyKIT will be compatible with 32-bit and 64-bit, python 2.7 and python 3.x
and then no matter what version of python is being run, the socket server
would never change, whether you're using 32-bit or 64-bit Unity3D, it would work.
